### DISCUSSION TITLE
Добавить в документацию

### DISCUSSION THREAD

**Discussion by Mr-Coquin** [moonshine-software/moonshine#1054] [General]
Добрый день. Столкнулся со следующим моментам:
Есть документация https://moonshine-laravel.com/docs/resource/models-resources/resources-events?change-moonshine-locale=ru 

И конечно первое что я пытался скажем переопределить данные это таким образом в ресурсе:
    protected function beforeCreating(Model $item): Model
    {
        $item->title = "Новый заголовок";
        return $item;
    }
    На что получал ошибку title null 
Полазив в Demo я увидел как вы это делаете через request()->merge
    protected function beforeCreating(Model $item): Model
    {
        request()->merge([
            'title' => "Новый заголовок",
        ]);
        return $item;
    }
И тогда все заработало. Возможно стоит для удобства сразу это включить в документацию ?

**Comment by lee-to** [comment]
Спасибо! Постараемся добавить
